### ISSUE TITLE
Launch the greeter in session for unlock

### ISSUE THREAD

**Issue by cavalier38** [the-cavalry/light-locker#22]
Switching VT has downsides. Try to reduce VT switching by launching the greeter in session and handling the authentication in light-locker.

**Comment by akostadinov** [comment]
Yes, switching VT is very ugly. gnome-screensaver does it on same session. With current implementation there are several shortcomings that made me stop using light-locker in favor of gnome-screensaver.
* When I switch to a different tty, lock is automatically triggered (so I can't see my log on tty12 when I want)
* When I lock screen, then if I later close lid, the laptop does not suspend; after unlock I see a password prompt for suspending. I guess because nobody is on the original VT, polkit wants a password for suspending and that cannot be provided by user
* switching VTs is ugly for the eye and multi-monitor settings are different

**Comment by jarnos** [comment]
> * When I switch to a different tty, lock is automatically triggered (so I can't see my log on tty12 when I want)

Not an issue here, but with nvidia driver lock-screen will not display in one laptop, and it will not display other TTYs in general there.

> * When I lock screen, then if I later close lid, the laptop does not suspend; after unlock I see a password prompt for suspending. I guess because nobody is on the original VT, polkit wants a password for suspending and that cannot be provided by user

Not an issue here.

> * switching VTs is ugly for the eye and multi-monitor settings are different

Not tested here

> -- light-locker 1.7.0

light-locker 1.8.0, Xubuntu 18.04
